task_sdk/pyproject.toml

```diff
@@ -27,10 +27,10 @@ dependencies = [
     "httpx>=0.27.0",
     "jinja2>=3.1.4",
     "methodtools>=0.4.7",
-    "msgspec>=0.18.6",
     "psutil>=6.1.0",
     "structlog>=24.4.0",
     "retryhttp>=1.2.0",
+    "msgspec>=0.19.0",
```

Comment:
Nit: sorting please

Reply:
We should add tomlsort somewhere in a pyproject.

Reply:
Addressed in https://github.com/apache/airflow/pull/45962/commits/edf815b0ae5b6b307133864d94f4435f57a3319e